src/og/control/CompassButton.js

```diff
@@ -86,6 +86,11 @@ let svg = `<?xml version="1.0" encoding="UTF-8" standalone="no"?>
  * @params {Object} [options] - Control options.
  */
 class CompassButton extends Control {
+
+    /**
+     * 
+     * @params {Object} [options] - Control options.
```

Comment:
...the same question. When class extends a class parameters are doubled. Is it ok?